tests/testthat/test-cache-low-level-api.R

```diff
@@ -1,16 +1,16 @@
 test_that("caching utils make right blocks with semi-colon", {
   blocks_simple_uncached <- compute_parse_data_nested(c("1 + 1", "2; 1+1")) %>%
-    dplyr::mutate(is_cached = FALSE) %>%
+    mutate(is_cached = FALSE) %>%
```

Comment:
where's `%>%` coming from if not dplyr?

Reply:
nvm :)

Reply:
{magritrr}? We'll at some  point transition to `|>` if minimal supported R version has it.